CHANGELOG_V1.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [v1.0.76](https://github.com/stargate/stargate/tree/v1.0.76) (2023-04-26)
+
+[Full Changelog](https://github.com/stargate/stargate/compare/v1.0.74.1...v1.0.76)
```

Comment:
v1.0.74.1 -> v1.0.75 (auto-generation sees tag for former but there's nothing in release notes for that one)

Reply:
Actually never mind, if that's what v1 had it should come as is.

Reply:
@tatu-at-datastax It showed the difference in change log, but both v1 and v2 had those PR information in the file, so I manually resolved it. Should I do a different PR with all the changes?